Frontend/src/app/components/header/header.component.spec.ts

```diff
@@ -1,21 +1,22 @@
 import {ComponentFixture, TestBed} from '@angular/core/testing';
 
 import {HeaderComponent} from './header.component';
-import {RouterTestingModule} from "@angular/router/testing";
-import {MatMenuModule} from "@angular/material/menu";
-import {Router} from "@angular/router";
+import {RouterTestingModule} from '@angular/router/testing';
+import {MatMenuModule} from '@angular/material/menu';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Mueller-Patrick/Betterzon/pullRequest?prid=7401082)
```suggestion
import {MatMenuModule} from "@angular/material/menu";
```